crates/biome_css_analyze/src/lint/nursery/no_unknown_property.rs

```diff
@@ -0,0 +1,86 @@
+use biome_analyze::{context::RuleContext, declare_rule, Ast, Rule, RuleDiagnostic};
+use biome_console::markup;
+use biome_css_syntax::CssGenericProperty;
+use biome_rowan::{AstNode, TextRange};
+
+use crate::utils::{is_kown_properties, vendor_prefixed};
+
+declare_rule! {
+    /// Disallow unknown properties.
+    ///
+    /// This rule considers properties defined in the `CSS Specifications and browser specific properties` to be known.
+    /// https://github.com/known-css/known-css-properties#source
+    ///
+    ///
+    /// This rule ignores:
+    ///
+    /// variables ($sass, @less, --custom-property)
+    /// vendor-prefixed properties (e.g., -moz-align-self, -webkit-align-self)
```

Comment:
```suggestion
    /// This rule ignores:
    ///
    /// - custom variables e.g. `--custom-property`
    /// - vendor-prefixed properties (e.g., `-moz-align-self,` `-webkit-align-self`)
```

We don't support sass and less